tests/attestation/attestation.bicepparam

```diff
@@ -10,4 +10,4 @@ param ccePolicies={
   attestation: ''
 }
 param managedIDName='cacisidecars'
-param attestationEndpoint='https://confidentialsidecars.weu.attest.azure.net'
+param attestationEndpoint=''
```

Comment:
Do we need to have the param already existing in this file or can we use the test.py file to insert the whole line instead of having a placeholder? Not sure which is better tbh, but this isn't a dealbreaker for me so merge if this way is better.

Reply:
It could be committed, but it would be set at runtime either way based on the env variable